app/helpers/vacancies_helper.rb

```diff
@@ -128,4 +129,24 @@ def vacancy_job_location(vacancy)
 
     address_join([organisation.name, organisation.town, organisation.county])
   end
+
+  def vacancy_job_location_heading(vacancy)
+    return I18n.t("school_groups.job_location_heading.#{vacancy.job_location}") unless
+      vacancy.job_location == 'at_multiple_schools'
+
+    if vacancy.parent_organisation.group_type == 'local_authority'
+      I18n.t('school_groups.job_location_heading.at_multiple_schools', organisation_type: 'local authority')
+    else
+      I18n.t('school_groups.job_location_heading.at_multiple_schools', organisation_type: 'trust')
+    end
```

Comment:
This could replace the if-else-end:
```
I18n.t('school_groups.job_location_heading.at_multiple_schools', organisation_type: organisation_type_basic(vacancy.parent_organisation))
```